config/app/manifest/en_US.json

```diff
@@ -1,12 +1,116 @@
+                "data": [
+                    "Your app advantage #1",
+                    "Your app advantage #2",
+                    "Your app advantage #3"
+                ]
+            },
+            "AsMuchAsYouNeed_ThisIsOptional": {
+                "title": "The title of another text section",
+                "type": "text",
+                "data": "You can add as much content as you need. It can also be a list, just change the 'type' property and use an array here as shown above."
+            }
+        },
+        "Legal": {
+            "LegalText_ThisIsOptional": {
+                "title": "A text with legal content",
+                "type": "text",
+                "data": "Free text with legal content."
+            },
+            "AsMuchAsYouNeedToo_ThisIsOptional": {
+                "title": "The title of another section",
+                "type": "text",
+                "data": "You can add as much content as you need. It can also be a list, just change the 'type' property and use an array here as shown above."
+            }
+        }
+    },
+    "assets": [
+        {
+            "type": "icon",
+            "url": "\/assets\/images\/app_name\/logo.svg"
+        },
+        {
+            "type": "image",
+            "url": "\/assets\/images\/app_name\/gallery\/app_picture_1.jpeg"
+        },
+        {
+            "type": "image",
+            "url": "\/assets\/images\/app_name\/gallery\/app_picture_1.png"
+        },
+        {
+            "type": "video",
+            "url": "https:\/\/wistia.com\/only-support-wistia-videos"
+        }
+    ],
+    "label": [
+        "** CHOOSE ONE OR MORE BELOW **",
+        "Silver Partner",
+        "Gold Partner",
+        "New",
+        "Popular",
+        "Free Trial"
+    ],
+    "resources": [
+        {
+            "title": "Homepage",
+            "url": "https:\/\/url-to-app-homepage.com",
+            "type": "homepage"
+        },
+        {
+            "title": "The 'type' property changes the resource icon on AppCatalog",
+            "url": "https:\/\/url-to-app-homepage.com\/user-documentation",
+            "type": "user-documentation"
+        },
+        {
+            "title": "A PDF file (the optional 'fileType' property makes it open inside AppCatalog without redirect)",
+            "url": "https:\/\/url-to-app-homepage.com\/its-possible-to-use-pdf-files.pdf",
+            "type": "developer-documentation",
+            "fileType": "pdf"
+        },
+        {
+            "title": "A Markdown file (the optional 'fileType' property makes it open inside AppCatalog without redirect)",
+            "url": "https:\/\/url-to-app-homepage.com\/its-possible-to-use-md-files.md",
+            "type": "release-notes",
+            "fileType": "markdown"
+        },
+        {
+            "title": "App internal doc (MAY be a pdf or markdown with the 'fileType' property)",
+            "url": "https:\/\/url-to-app-homepage.com\/internal-documentation",
```

Comment:
Why do we need to use `\/` can you please fix the code so we can use slashes without escaping them?

Reply:
that's weird.. I created them using the manifest creation command.. I'll check what's going on